### ISSUE TITLE
Nanny Cam web site not loading, QR Code reader not working

### ISSUE THREAD

**Issue by aerialviews007** [eried/FlipperZeroMayhem#54]
Installed the release and the nanny cam web site does not load.  I also cannot get the QR code reader to pick up anything.  Are these functions dead?

**Comment by eried** [comment]
No, they should work. What is the QR displaying? Does the camera works ok?

**Comment by aerialviews007** [comment]
The QR FAP shows “waiting for QR code. 
the nannycam app shows the proper information for the SSID but the website doesn’t come up. I e tried multiple devices including adding http:// before. 
Camera, Morse code and Motion all work great though.

**Comment by eried** [comment]
Since the resolution of the camera is very low, I suggest to you to try a bigger QR :) 

For the nannycam, could you try another phone? or maybe in your PC? I have seen reports of issues with some devices, mostly iphones.

**Comment by aerialviews007** [comment]
Okay, I was able to get the QR code reader to work with a larger QR code.

Nannycam is not responding to HTTP requests.  I checked with wget.  "Read Error (Connection reset by peer) in headers"

**Comment by eried** [comment]
Some browsers struggle with nannycam, but I haven't found the reason. It works on my android phone so if you can test and give me more info maybe I could fix it
